services/QuillLMS/engines/evidence/app/models/evidence/prompt_response.rb

```diff
@@ -18,20 +18,38 @@ class PromptResponse < ApplicationRecord
     DIMENSION = 1536
     MODEL = 'text-embedding-3-small'
 
+    DISTANCE_METRIC = 'cosine'
+
     belongs_to :prompt
+    has_one :prompt_response_feedback, dependent: :destroy
 
     has_neighbors :embedding
 
-    validates :text, presence: true
+    validates :response_text, presence: true
     validates :embedding, presence: true
     validates :prompt, presence: true
 
     before_validation :set_embedding
 
+    def closest_prompt_response
+      @closest_prompt_response ||= begin
```

Comment:
This is probably fine to memoize give our use case, but this is a method whose value isn't constant and can change over time.

Reply:
That's a good point.  I was thinking of memoization to prevent duplicate calls for closest feedback, but I can just use a temp variable there.